copyhtml/copyhtml.ts

```diff
@@ -0,0 +1,9 @@
+import { getPromptResult } from "./prompt.ts";
+import { scrape } from "./scrape.ts";
+import { setClipboardContent } from "./clipboard.ts";
```

Comment:
Nit: The helper functions are all small enough to be contained within this file.

Reply:
Resolved by <https://github.com/EthanThatOneKid/copyhtml/pull/1/commits/65cc1b38a4c61926775cb45fdba705d7cc921cf5>.